introspection.go

```diff
@@ -4,9 +4,11 @@ import (
 	"fmt"
 	"reflect"
 	"sort"
+	"strings"
 
 	"github.com/dagger/graphql/language/ast"
 	"github.com/dagger/graphql/language/printer"
+	"golang.org/x/exp/slices"
```

Comment:
Looks like this either needs a `go mod tidy` or an upgrade of go version, so we can use https://pkg.go.dev/slices (which is now built-in to the stdlib 🎉)